src/backend/CassandraBackend.cpp

```diff
@@ -1386,6 +1461,28 @@ CassandraBackend::open(bool readOnly)
         if (!executeSimpleStatement(query.str()))
             continue;
 
+        query.str("");
+        query << "CREATE INDEX IF NOT EXISTS " << tablePrefix
+              << "index_issuer_nf_tokens_taxon ON " << tablePrefix
+              << "issuer_nf_tokens (issuer, token_taxon)";
+        if (!executeSimpleStatement(query.str()))
+            continue;
```

Comment:
We should not have an index. Indexes in Cassandra have serious performance implications, because querying the index requires contacting every node in the cluster.